traits/src/lib.rs

```diff
@@ -111,3 +111,10 @@ pub trait BasicCurrencyExtended<AccountId>: BasicCurrency<AccountId> {
 	/// Add or remove abs(`by_amount`) from the balance of `who`. If positive `by_amount`, do add, else do remove.
 	fn update_balance(who: AccountId, by_amount: Self::Amount) -> result::Result<(), Self::Error>;
 }
+
+/// Oracle feed data event
```

Comment:
```suggestion
/// When new data is received
```

`OnNewData` can be very generic and used modules other than oracle